.orchestra/support/revng-distributable/install.sh

```diff
@@ -0,0 +1,125 @@
+grep -Ev -e '^revng/' \
+         -e '^share/revng/qa/' \
+         -e '^lib64/libclang.*\.so' \
+         -e 'lto-dump' \
+         -e '\.mypy_cache' -e '__pycache__' | \
+grep -E -e "^bin/" \
+        -e "^include/revng/PipelineC/" \
+        -e "^lib$" \
+        -e "^lib64/python/" \
+        -e "^lib64/[^/]*\.(bc|so)$" \
+        -e "^lib64/[^/]*\.so\.[^/]*$" \
+        -e "^lib64/gcc/x86_64-pc-linux-gnu/11.2.0/[^/]*\.(o|a)$" \
+        -e "^lib64/llvm/clang-release/lib/[^/]*\.so\.[^/]*$" \
+        -e "^lib64/llvm/clang-release/bin/llc$" \
+        -e "^lib64/llvm/clang-release/bin/llvm-link$" \
+        -e "^lib64/revng/analyses/[^/]*\.so$" \
+        -e "^libexec/gcc/x86_64-pc-linux-gnu" \
+        -e "^libexec/revng/" \
+        -e "^share/orchestra/" \
+        -e "^share/revng/pipelines/" \
+        -e "^share/revng/[^/]*\.ll$" \
+        -e "^x86_64-pc-linux-gnu/" | \
+sed 's|^|/|' > "$FILE_LIST"
+
+rsync \
+  --archive \
+  --verbose \
+  --progress \
+  --files-from="$FILE_LIST" \
+  "$ORCHESTRA_ROOT/." \
+  "${TMP_ROOT}${ORCHESTRA_ROOT}/revng/root/."
+
+cd "$TMP_ROOT/$ORCHESTRA_ROOT/revng"
+
+echo "Creating environment"
+cp -a "$ORCHESTRA_DOTDIR/support/revng-distributable/environment" environment
+
+# shellcheck disable=SC2094
+orchestra environment | \
+  grep -E "^export (PATH|PKG_CONFIG_PATH|PYTHONPATH|LIBRARY_PATH)" \
+    >> environment
+
+cat >> environment <<EOF
+unset ORCHESTRA_DOTDIR
+unset ORCHESTRA_ROOT
+EOF
+
+echo "Copying README.md"
+cp -a "$ORCHESTRA_DOTDIR/support/revng-distributable/README.md" .
+
+echo "Copying revng"
+cp -a "$ORCHESTRA_DOTDIR/support/revng-distributable/revng" revng
+
+
+cd "$TMP_ROOT/$ORCHESTRA_ROOT/revng/root"
+
+echo "Copying revng-distributable scripts"
+cp -a \
+  "$ORCHESTRA_DOTDIR/support/revng-distributable/revng-update" \
+  "$ORCHESTRA_DOTDIR/support/revng-distributable/revng-system-info" \
+  bin/
+
+echo "Copying install-revng-dependencies"
+cp -a "$ORCHESTRA_DOTDIR/support/revng-distributable/dockers/install-revng-dependencies" bin/
+
+echo "revng-distributable updater info"
+mkdir -p share/revng-distributable
+echo 1 > share/revng-distributable/version
+
+
+echo "Stripping components"
+STRIP_FILE_LIST=$(mktemp -p "${TMP_ROOT}")
+find 'share/orchestra' -type f -name '*.idx' ! -name '*revng*' -exec cat {} \; > "$STRIP_FILE_LIST"
+find 'libexec/revng' -type f >> "$STRIP_FILE_LIST"
+while read -r FILE; do
+  if [ -f "$FILE" ] && file "$FILE" | grep -qE 'ELF.*x86-64.*(shared|dynamic).*not stripped'; then
+    echo strip "$FILE"
+    strip "$FILE"
+  fi
+done < "$STRIP_FILE_LIST"
+
+echo "Fix .idx"
+ALL_FILES=$(find . -type f | sed 's;^\.\/;;g' | sort)
+TMP_IDX=$(mktemp -p "${TMP_ROOT}")
+for IDX in share/orchestra/*.idx; do
+  comm -12 <(sort "$IDX") <(echo "$ALL_FILES") > "$TMP_IDX"
+  if [ "$(wc -l < "$TMP_IDX")" -gt 3 ]; then
+    sort "$TMP_IDX" > "$IDX"
+  else
+    rm -f "$IDX" "${IDX//.idx/.license}" "${IDX//.idx/.json}"
+  fi
+done
```

Comment:
What's going on here?
We can't drop the license files from the release.

Reply:
Basically it's rewriting the `.idx` files, trimming away any file that has been removed
The `rm` is in the case _all_ the files of the component have been deleted, to keep things tidy